presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlTypeMapping.java

```diff
@@ -219,6 +239,214 @@ public void testDecimalExceedingPrecisionMax()
         testUnsupportedDataType("decimal(50,0)");
     }
 
+    @DataProvider
+    public Object[][] testDecimalExceedingPrecisionMaxProvider()
+    {
+        return new Object[][] {
+                {60, 20},
+                {65, 25},
+        };
+    }
+
+    @Test(dataProvider = "testDecimalExceedingPrecisionMaxProvider")
+    public void testDecimalExceedingPrecisionMaxWithUnsupportedValues(Integer typePrecision, Integer typeScale)
+    {
+        JdbcSqlExecutor jdbcSqlExecutor = new JdbcSqlExecutor(mysqlServer.getJdbcUrl());
+
+        List<String> testData = asList(
+                 "1234567890123456789012345678901234567890.1234567890",
+                "-1234567890123456789012345678901234567890.1234567890");
+        int largestScale = testData.stream().map(e -> e.length() - max(e.indexOf("."), 0) - 1).max(Integer::compareTo).orElse(0);
```

Comment:
please format this as:
```
int largestScale = testData.stream()
    .map(e -> e.length() - max(e.indexOf("."), 0) - 1)
    .max(Integer::compareTo)
    .orElse(0);

```

rename `e` to `value`

In general try to format chained code this way.

The best would be to simply use `10` :)

Same comments for below and other places like that.